### PR TITLE
Fix import

### PR DESCRIPTION
**Pull request by joel**

### CODE REVIEW

File: lib/dir_model/import.rb [hunk 1]
```diff
@@ -45,22 +45,21 @@ def next(path, context={}, previous=nil)
 
     def match?
       return if load_state == :loaded
+      @_match = loader { find_match }
```
Comment (s12chung) [inline lib/dir_model/import.rb:48]: `@_match ||=` ?

**Comment by s12chung** [comment]
:+1:

File: lib/dir_model/import.rb (hunk 1 above)
Comment (joel) [inline lib/dir_model/import.rb:48]: In `find_match` I store if there is a match, true or false in `@_match` and all informations about matches into `@file_infos` so in `skip?` I use :  

```
    def skip?
      load
      !@_match
    end
```

and in `matches` 

```
    def matches
      load
      file_infos[:options][:match]
    end
```

Comment (s12chung) [inline lib/dir_model/import.rb:48]: ah.. i see.

**Pull request merged**